about.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/nav-bar.css">
+    <link rel="stylesheet" href="./styles/about-styles.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <title>Global Training Courses - About Us</title>
+</head>
+<body>
+  <header>
+    <nav class="nav-mobile">
+      <img class="menu-button" id="menu-button" src="./assets/hamburger-menu.svg" alt="menu">
+      <div class="menu" id="menu">
+        <div>
+          <img id="close-button" src="./assets/close.png" alt="close">
+        </div>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="index.html#topics">Topics</a></li>
+          <li><a href="index.html#courses">Courses</a></li>
+          <li><a href="#">Log In</a></li>
+          <li><a href="#">Sign Up</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="nav-desktop">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#topics">Topics</a></li>
+        <li><a href="#courses">Courses</a></li>
+        <li><a href="#">Log In</a></li>
+        <li><a href="#">Sign Up</a></li>
+      </ul>
+    </nav>
+    <h1>GLOBAL TRAINING COURSES</h1>
+    <p class="about-info">Global Training Courses is a worldwide leader in IT and technology training, helping individuals and organizations build the skills they need to succeed in an ever-changing world.<br><br>
+      With an international network of offices and training facilities, Global Knowledge has the unique flexibility to deliver a broad portfolio of courses on over 100 countries online, in classrooms, and through a worldwide partner network.<br><br>
+      Established in 1995, Global Training Courses has over 1000 employees worldwide and enables the success of more than 200,000 tech professionals each year.<br><br>
+      In June 2021, Global Training Courses merged with Skillsoft to create the world’s leading corporate learning company.</p>
+    <div class="contact-info">
+      <p>Customer support, sales assistance and enterprise clients available.</p>
+      <p>1-800-268-7737</p>
+      <p>contact@skillsoft.com</p>
+    </div>
+  </header>
+  <main>
+    <div class="facts">
+      <h3>Quick Facts:</h3>
+      <div class="border-topic"></div>
```

Comment:
please try to add a section that includes images in mobile and desktop versions as the Design provided 
![image](https://user-images.githubusercontent.com/61790587/164915049-0e489460-2c2e-48e4-a00f-328680cb8ccf.png)

Reply:
- [ ] This section is still missing, kindly fix this, also your about page footer should be dark on desktop view and white on mobile view according to the project requirement, see the image below to guide you
![Screenshot from 2022-04-24 00-48-24](https://user-images.githubusercontent.com/66097371/164949820-1768775e-926a-473e-928d-13882fd8aa69.png)

Reply:
- [ ] As requested by the previous reviewers, you should add the `See the past` section to the about page. The section is still missing. Following the project requirements, you must stick to the original design as much as you can (layouts, margins ...). On the about page, the original design has 5 sections: Header, about section, Logo section, see the past section, and the footer. Make sure that your design has the logo section and "see the past section" and try to stick to the original design as much as you can:
- Original design:
<img width="488" alt="image" src="https://user-images.githubusercontent.com/77190203/164954799-9e15ea59-b2a4-43f9-b632-ade39326395a.png">

- Your design:
<img width="793" alt="image" src="https://user-images.githubusercontent.com/77190203/164954864-2365027a-b5bc-4be4-8cf6-714c1d9b9b79.png">